docs/4.3/aws_oss_guide.md

```diff
@@ -386,12 +379,308 @@ To upgrade to a newer version of Teleport:
 - Copy  and its contents should also be backed up and copied over to the new instance.
 - Either restart the instance, or log in via SSH and run `sudo systemctl restart teleport.service`
 
-# Running Teleport Enterprise on AWS
-Most of this guide has been designed for OSS Teleport. Most of this guide also applies to Teleport Enterprise
-with a few extra notes around adding a license and getting the correct binary. If you would
-like help setting up Teleport Enterprise on AWS, please mail us at info@gravitational.com
+## Accessing EKS using Teleport
+
+This guide is based on an orignial post from AWS Open Source Blog - [Authenticating to EKS Using GitHub Credentials with Teleport](https://aws.amazon.com/blogs/opensource/authenticating-eks-github-credentials-teleport/) but will be updated with latest Teleport version & best practices.
+
+### Prerequisites
+You’ll need a functioning EKS cluster, we recommend version 1.16.  If you’re unfamiliar
+with creating an EKS cluster, see [eksctl.io](https://eksctl.io). Make sure you have eksctl version 0.22.0.
```

Comment:
Link to https://aws.amazon.com/eks/getting-started/ here instead.

Reply:
Added